docs/config.md

```diff
@@ -643,9 +646,9 @@ perf_analyzer_flags:
 
 **Important Notes**: 
 * The Model Analyzer also provides certain arguments to the `perf_analyzer`
-  instances it launches. These ***cannot*** be overriden by providing those
-  arguments in this section. An example of this is `perf_measurement_window`,
-  which is an argument to Model Analyzer itself.
+  instances it launches. An example of this is `perf_measurement_window`,
+  which is an argument to Model Analyzer itself. In some cases, those ***cannot*** be overriden by providing those arguments in this section, but in most cases, they
```

Comment:
Could you be more specific about "some cases" and "most cases"? Also, keep the number of characters in each line below 80.